component/mcsd/component_test.go

```diff
@@ -604,3 +604,108 @@ func TestGetLastUpdated(t *testing.T) {
 		})
 	}
 }
+
+func TestComponent_ExcludeAdminDirectories(t *testing.T) {
```

Comment:
convention is to name tests as the function you're testing, so `TestComponent_registerAdministrationDirectory`